appengine-java11/helidon-mp-gae-greet/README.md

```diff
@@ -0,0 +1,192 @@
+```
+
+Open a new browser tab and http://localhost:8080/greet (i'm using a different port). *You may see a message "No handler found for path: /" if you go to http://localhost:8080, which is okay. we have not defined that.*
+
+![helidon-mp-example-gae-localhost](https://user-images.githubusercontent.com/902972/82602591-f1005b00-9b76-11ea-872a-266392d73ca1.png "Sample page for 'greet'resource")
+
+# Part 2: Preparing for Google App Engine
+
+Helidon app may run only on [Java 11 runtime](https://cloud.google.com/appengine/docs/standard/java11) (i haven't tried it on Java 8 runtime).
+
+The primary requirement for the application to be deployed to Google app engine is a yaml file.
+
+## 1 - Remove the default `app.yaml` under the project root.
+
+## 2 - Add `helidon-mp-app.yaml` in a new folder, like gae under `src/main`
+
+```
+helidon-mp-gae-greet
+|_src
+  |_main
+    |_gae
+      |_helidon-mp-app.yaml
+```
+
+Add the below to `helidon-mp-app.yaml`
+```
+runtime: java11
+entrypoint: java -Xmx64m -jar ${project.artifactId}.jar
+```
+
+## 3 - The below is added to `pom.xml` (i know this maybe frustrating). Don't worry about all the details, this allows the helion-mp-gae.yaml file to be copied to `/target` folder.
+
+```
+...
+<build>
+  <plugins>
+    <plugin>
+     ...
+    <plugin> 
+    
+    # --add from here..
+    <plugin>                    
+      <groupId>org.apache.maven.plugins</groupId>                  
+      <artifactId>maven-resources-plugin</artifactId>                 
+      <executions>
+        <execution>
+          <id>copy-gae-file</id>
+          <phase>process-resources</phase>                    
+          <goals>
+            <goal>copy-resources</goal>
+          </goals>
+          <configuration>                            
+            <outputDirectory>${project.build.directory}</outputDirectory>
+            <resources>
+              <resource>                                    
+                <filtering>true</filtering>
+                <directory>src/main/gae</directory>
+                <includes>                                         
+                  <include>helidon-mp-app.yaml</include>
+                  <include>.gcloudignore</include>
+                </includes>
+              </resource>
+            </resources>
+          </configuration>
+        </execution>
+      </executions>
+    </plugin>
+    # --up to here 
+</build>
+```
+
+## 4 - Rebuild the project
+```
+$ mvn clean package
+```
+
+## 5 - Setup Google App Engine
+
+Create a new Project on Google Cloud. Create New App, Choose `Java` as Language and `Standard Environment`.
+
+
+
+## 6 - Deploy to Google App Engine
+
+Go to your project root and run this command.
+
+*Tip: You will need Google Cloud SDK installed on your machine. I'm running this on my Mac. Run gcloud -v to make sure gcloud is installed.*
+
+```
+$ gcloud app deploy ./target/helidon-mp-app.yaml --project=<gcp-project-name>
```

Comment:
```suggestion
$ gcloud app deploy ./target/helidon-mp-gae-greet.jar
```

Reply:
This `--project=<gcp-project-name>` will be required if user has multiple projects on their account.